services/configuration/src/tests/api.rs

```diff
@@ -286,6 +293,50 @@ fn test_votes_for_propose() {
     }
 }
 
+#[test]
+fn test_dissenting_votes_for_propose() {
+    use schema::VotingDecision;
+
+    let mut testkit: TestKit = TestKit::configuration_default();
+    let api = testkit.api();
+    // Apply a new configuration.
+    let cfg_change_height = Height(5);
```

Comment:
nitpick: I don't think it's necessary to use additional `cfg_change_height` variable here - you can use `set_actual_from(Height(5))`

Reply:
Done